tests/test_1477_custom_interpretation.py

```diff
@@ -0,0 +1,96 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/uproot5/blob/main/LICENSE
+
+import re
+
+import numpy
+import pytest
+import skhep_testdata
+
+import uproot
+import uproot.interpretation.custom
+
+awkward = uproot.extras.awkward()
+pandas = uproot.extras.pandas()
```

Comment:
```suggestion
awkward = pytest.importorskip("awkward")
pandas = pytest.importorskip("pandas")
```

Thank you @mrzimu! Just one last minor change to make the tests pass.